internal/auth0/page/resource.go

```diff
@@ -0,0 +1,232 @@
+						},
+						"show_log_link": {
+							Type:        schema.TypeBool,
+							Required:    true,
+							Description: "Indicates whether to show the link to logs as part of the default error page. Defaults to `true`.",
+						},
+						"url": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							ValidateFunc: internalValidation.IsURLWithHTTPSorEmptyString,
+							Description:  "URL to redirect to when an error occurs, instead of showing the default error page.",
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func createPages(ctx context.Context, data *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	data.SetId(id.UniqueId())
+	return updatePages(ctx, data, meta)
+}
+
+func readPages(_ context.Context, data *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	api := meta.(*config.Config).GetAPI()
+
+	globalClientID, err := fetchGlobalClientID(api)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	clientWithLoginPage, err := api.Client.Read(globalClientID)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	tenantPages, err := api.Tenant.Read()
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	result := multierror.Append(
+		data.Set("login", flattenLoginPage(clientWithLoginPage)),
+		data.Set("change_password", tenant.FlattenTenantChangePassword(tenantPages.GetChangePassword())),
+		data.Set("guardian_mfa", tenant.FlattenTenantGuardianMFAPage(tenantPages.GetGuardianMFAPage())),
+		data.Set("error", tenant.FlattenTenantErrorPage(tenantPages.GetErrorPage())),
+	)
+
+	return diag.FromErr(result.ErrorOrNil())
+}
+
+func updatePages(ctx context.Context, data *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	api := meta.(*config.Config).GetAPI()
+
+	if clientWithLoginPage := expandLoginPage(data); clientWithLoginPage != nil {
+		globalClientID, err := fetchGlobalClientID(api)
+		if err != nil {
+			return diag.FromErr(err)
+		}
+
+		if err := api.Client.Update(globalClientID, clientWithLoginPage); err != nil {
+			return diag.FromErr(err)
+		}
+	}
+
+	if tenantPages := expandTenantPages(data.GetRawConfig()); tenantPages != nil {
+		if err := api.Tenant.Update(tenantPages); err != nil {
+			return diag.FromErr(err)
+		}
+	}
+
+	return readPages(ctx, data, meta)
+}
+
+func deletePages(_ context.Context, _ *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	api := meta.(*config.Config).GetAPI()
+
+	globalClientID, err := fetchGlobalClientID(api)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	if err := api.Client.Update(globalClientID, &management.Client{
+		CustomLoginPageOn: auth0.Bool(false),
+	}); err != nil {
+		return diag.FromErr(err)
+	}
+
+	if err := api.Tenant.Update(&management.Tenant{
+		ChangePassword: &management.TenantChangePassword{
+			Enabled: auth0.Bool(false),
+		},
+		ErrorPage: &management.TenantErrorPage{
+			ShowLogLink: auth0.Bool(false),
+			URL:         auth0.String(""),
```

Comment:
Maybe clear out the HTML property too?

Reply:
Should this be done for the HTML properties of `login`, `change_password`, and `guardian_mfa` as well?

Reply:
As long as those pages are disabled, setting the HTML as well to empty is unnecessary. We preferred doing this as a user could in the future come back to these pages and re-enable them with the previous HTML.